os/android/utils_android.h

```diff
@@ -118,6 +118,7 @@ static uint32_t DrmFormatToHALFormat(int format) {
     case DRM_FORMAT_XBGR8888:
     case DRM_FORMAT_RGBX8888:
     case DRM_FORMAT_ABGR8888:
+      return HAL_PIXEL_FORMAT_RGBA_8888;
```

Comment:
this seems wrong, for all of these formats we return same format?

Reply:
I will update. Thanks